batch/scripts/_add_gfw_fields_sql.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
I really like that you separated these out into subscripts, but I wonder if there's an established pattern or naming convention for doing that with bash scripts? The "_" in front of the script feels more like a python thing, although works

Reply:
A quick googling makes me think we have a pretty uncommon use case of using bash scripts for an actual processing pipeline instead of one-off system admin scripts... so maybe just do as you like!

Reply:
Yeah, breaking these up was a really good idea! 🔨